.travis.yml

```diff
@@ -2,31 +2,31 @@ language: python
 
 sudo: false
 
-
 matrix:
   include:
     # Only one test for these Python versions
     - python: 3.5
-      env: PANDAS=0.20.2 MATPLOTLIB=1.5.3
+      env: ENV_FILE="ci/travis/35-minimal.yaml"
 
     # Python 2.7 and 3.6 test all supported Pandas versions
     - python: 2.7
-      env: PANDAS=0.19.2  MATPLOTLIB=1.5.3 SHAPELY=1.5
-    - python: 2.7
-      env: PANDAS=0.20.2  MATPLOTLIB=2.0.2 CHANNEL=conda-forge
+      env: ENV_FILE="ci/travis/27-pd020.yaml"
     - python: 2.7
-      env: PANDAS=0.23.2  MATPLOTLIB=2.1.2
+      env: ENV_FILE="ci/travis/27-latest-defaults.yaml"
     - python: 2.7
-      env: PANDAS=master  MATPLOTLIB=2.2.2
+      env: ENV_FILE="ci/travis/27-dev.yaml"
 
     - python: 3.6
-      env: PANDAS=0.19.2  MATPLOTLIB=1.5.3 SHAPELY=1.5 CHANNEL=conda-forge # conda-forge is needed here for this combination
+      env: ENV_FILE="ci/travis/36-pd020.yaml"
+    - python: 3.6
+      env: ENV_FILE="ci/travis/36-pd022.yaml"
+
     - python: 3.6
-      env: PANDAS=0.20.2  MATPLOTLIB=2.0.2
+      env: ENV_FILE="ci/travis/37-latest-defaults.yaml"
```

Comment:
The line above should be updated to `3.7` (though, of course, `3.7` is tested as specified in the env file, underlining your original point).

Reply:
I tried that first, but travis has no 3.7 yet. In any case, I mainly kept them as information to quickly see which python version are tested. I could also remove it entirely (as it is the environment file that determines the python version used in the tests)

Reply:
@ocefpaf What do you mean with "specify the python version in the yaml file"? Where in the `travis.yaml` file can you mention that?

Reply:
Ah, I suppose you meant the conda environment yaml files? (that would make more sense :))

So that is actually already the case. I originally only kept the `python: 3.6` in the `travis.yaml` file because they were already there, and to have a nicer view on the python versions used in the different builds on the travis web interface. 

But I should probably just remove them ..

Reply:
> Ah, I suppose you meant the conda environment yaml files? (that would make more sense :))

Yes. Sorry for the yaml confusion :grimacing: 

>  and to have a nicer view on the python versions used in the different builds on the travis web interface.

You can use Travis-CI yaml's matrix `name` section for that. More verbose, not as elegant, but it should have the same results.

Reply:
@jorisvandenbossche Ah, that makes sense. I find the version in your latest commit of what is being tested to be clearer to see in the overview on Travis.